themes/bootstrap5/js/facets.js

```diff
@@ -331,6 +333,41 @@ VuFind.register('multiFacetsSelection', function multiFacetsSelection() {
     }
   }
 
+  function getModifiedFiltersCount() {
+    return document.querySelectorAll('[data-multi-filters-modified="true"]').length;
+  }
+
+  function updateCountText() {
+    const textElems = document.getElementsByClassName('multi-filters-text');
+    const count = getModifiedFiltersCount();
+    const text = count === 0 ? defaultCountText : 'You have modified ' + count + ' filters.';
```

Comment:
We need to add internationalization support for this.

My suggestion would be to add this to languages/en.ini:

`modified_filter_count = "You have modified %%count%% filter(s)."`

Then register the new translation string in [js-translations.phtml](https://github.com/vufind-org/vufind/blob/dev/themes/bootstrap5/templates/layout/js-translations.phtml)

Then change replace the hard-coded text concatenation with:

`VuFind.translate('modified_filter_count', { '%%count%%': count })`

In an ideal world, it would be even better to take advantage of the ICU pluralization functionality of the translation system to be smarter about filter vs. filters, but unfortunately that only works in PHP, so for now at least, the filter(s) compromise is our best bet.